src/runtime/traceback.cpp

```diff
@@ -145,7 +145,8 @@ void setupTraceback() {
     NULL));
     */
     traceback_cls->giveAttrDescriptor("tb_next", traceback_tb_next, NULL);
-
+    traceback_cls->giveAttr("tb_lineno",
+                            new BoxedMemberDescriptor(BoxedMemberDescriptor::INT, offsetof(BoxedTraceback, tb_lineno)));
```

Comment:
You could probably do `offsetof(BoxedTraceback, line) + offsetof(LineInfo, line)` instead of storing the attribute twice.